Apps/WebClient/src/ClientApp/src/store/modules/waitlist/actions.ts

```diff
@@ -35,35 +35,67 @@ export const actions: WaitlistActions = {
                 });
         });
     },
+    scheduleCheckIn(
+        context,
+        params: {
+            getTicketOnFail: boolean;
+            ticket?: Ticket;
+        } = { getTicketOnFail: false }
+    ): Promise<void> {
+        const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+        const checkInTimeoutId = context.state.checkInTimeoutId;
+        if (checkInTimeoutId) {
+            logger.debug(
+                `Waitlist - Clearing checkIn timeout (${checkInTimeoutId}).`
+            );
+            clearTimeout(checkInTimeoutId);
+        }
```

Comment:
`clearTimeout` can be called with an invalid ID and it will just do nothing.

```suggestion
        clearTimeout(context.state.checkInTimeoutId);
```